app/src/main/assets/pages/Insuree.js

```diff
@@ -109,6 +109,7 @@ $(document).ready(function () {
                 var InsureeId = Android.SaveInsuree(jsonInsuree, FamilyId, 0, parseInt(ExceedThreshold), PolicyId);
                 if (PolicyId > 0 && TotalIns >= MemberCount) {
                     $("#divProgress").hide();
+                    window.open("FamilyAndInsurees.html?f=" + FamilyId, "_self");
                 } else {
                     $("#divProgress").hide();
                     window.open("FamilyAndInsurees.html?f=" + FamilyId, "_self");
```

Comment:
why not removing the if ? (DON'T COMMIT THIS AS THERE IS STILL THE CLOSING BRACKET)
```suggestion
                $("#divProgress").hide();
                window.open("FamilyAndInsurees.html?f=" + FamilyId, "_self");
```

Reply:
I think we can remove the if,
I didn't understand the block, that's
why I left it in
I didn't want to broke any process

Reply:
please remove the if and keep the inner code